src/definitions.jl

```diff
@@ -245,7 +246,13 @@ size(p::ScaledPlan) = size(p.p)
 show(io::IO, p::ScaledPlan) = print(io, p.scale, " * ", p.p)
 summary(p::ScaledPlan) = string(p.scale, " * ", summary(p.p))
 
-*(p::ScaledPlan, x::AbstractArray) = scale!(p.p * x, p.scale)
+if VERSION >= v"0.7.0-DEV.3665"
+    *(p::ScaledPlan, x::AbstractArray) = LinearAlgebra.rmul!(p.p * x, p.scale)
```

Comment:
Could also bump the (second) lower bound in REQUIRE to `0.7.0-DEV.3665` and do this unconditionally. Opinions?